front/lib/api/assistant/generation.ts

```diff
@@ -112,9 +112,9 @@ export async function renderConversationForModel({
       });
     } else if (isContentFragmentType(m)) {
       messages.unshift({
-        role: "content_fragment" as const,
-        name: "inject_content_fragment",
```

Comment:
we can't change that as content_fragment is turned into a function which hints the model that it was the return of an hypotethical function called "inject_content_fragment"